FrontEnd/src/components/basicPage/BasicPage.jsx

```diff
@@ -69,29 +67,41 @@ function BasicPage() {
         },
       }}
     >
-      <Header isAuthorized={auth.isAuth} />
+      <SWRConfig value={{
+        onError: (error) => {
+          console.error(error);
+          if (error.status === 401) {
```

Comment:
nit: to avoid magic numbers you might consider using https://www.npmjs.com/package/http-status-codes